basics/Main.java

```diff
@@ -0,0 +1,72 @@
+import java.time.LocalDateTime;
+
+public class Main {
+    
+public static void main(String[] args) {
+int dogCount = 1;
+System.out.println("I own " + dogCount + " " + pluralize("dog", dogCount) + ".");
+
+int catCount = 2;
+System.out.println("I own " + catCount + " " + pluralize("cat", catCount) + ".");
+
+int turtleCount = 0;
+System.out.println("I own " + turtleCount + " " + pluralize("turtle", turtleCount) + ".");
+
+flipNHeads(1);
+
+Main object = new Main();
+object.clock();
+}
+
+
+
+public static String pluralize(String word, int number) {
+
+    if (number == 1) {
+    return word;
+    } else {
+    return word + "s";
+}
+}
+
+public static void flipNHeads(int n) {
+
+int counter = 0;
+int headCounter = 0;
+
+while (headCounter<n) {
+    counter++;
+if (Math.random() < 0.5){
+System.out.println("Tails");
+} else {
+System.out.println("Heads");
+headCounter ++;
+
+}
+}
+System.out.println("it took " + counter + " flips to flip " + n + " heads in a row");
```

Comment:
Your main method contains a call to just look for 1 head in a row, which means it's possible to miss a bug in your code! If we're looking for 3 heads in a row, this will finish even if the 3 heads aren't all in a row, e.g. if we get HTHTH. Make sure this method deals with that case! You'll probably need to think about your headCounter when you flip tails to solve this problem.